draft-ietf-quic-recovery.md

```diff
@@ -509,6 +509,57 @@ frames containing them could be lost. In this case, the loss recovery algorithm
 may cause spurious retransmits, but the sender will continue making forward
 progress.
 
+## Tracking Sent Packets {#tracking-sent-packets}
+
+QUIC stores information about every packet sent. It's expected implementations
+will index this per-packet information by packet number and store the
+per-packet fields detailed below for loss recovery and congestion control.
+Additionally, implementations MUST ensure that any retransmittable frames
+being transmitted are tracked in case of loss.
+
```

Comment:
"Implementations MUST track the ack blocks in a packet in order to stop sending those ack blocks if the packet is 
acknowledged."

Perhaps this is not strictly loss-recovery-related, but this is the section where I'm going to go to implement struct sent_pkt.